dataclients/kubernetes/testdata/ingressV1/traffic/three-backends-with-one-zero-weight.eskip

```diff
@@ -4,7 +4,8 @@ kube_namespace1__ingress1______:
 
 kube_namespace1__ingress1__test_example_org____service1v1:
   Host("^(test[.]example[.]org[.]?(:[0-9]+)?)$") &&
-  Traffic(0.3)
+  Traffic(0.3) &&
+  True()
```

Comment:
It also fixes this pre-existing test case.

Reply:
I don't really understand the comment, but yes you need to adjust if you simply the code like now. :)

Reply:
yes, this is what I meant